sentry-android-integration-tests/test-app-plain/build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+plugins {
+    id("com.android.application")
+}
+
+android {
+    compileSdk = 32
+
+    defaultConfig {
+        applicationId = "io.sentry.java.tests.perf.appplain"
+        minSdk = 21
+        targetSdk = 32
+        versionCode = 1
+        versionName = "1.0"
+    }
+
+    buildTypes {
+        getByName("release") {
+            isMinifyEnabled = true
+            signingConfig = signingConfigs.getByName("debug") // to be able to run release mode
+            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "benchmark-proguard-rules.pro")
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+    buildFeatures {
+        viewBinding = true
+    }
+    signingConfigs {
+        getByName("debug") {
+            storeFile = rootProject.file("debug.keystore")
+            storePassword = "android"
+            keyAlias = "androiddebugkey"
+            keyPassword = "android"
+        }
+    }
+
+    variantFilter {
+        if (Config.Android.shouldSkipDebugVariant(buildType.name)) {
+            ignore = true
+        }
+    }
+}
+
+dependencies {
```

Comment:
The Sentry Android Core lib has the dependencies of:

```
    // lifecycle processor, session tracking
    implementation(Config.Libs.lifecycleProcess)
    implementation(Config.Libs.lifecycleCommonJava8)
    implementation(Config.Libs.androidxCore)
```

But most if not all Apps also have them as well, what if we add those dependencies in the plain test so they are not counted for the diff in size?

Reply:
Hmm, personally I'm not sure about that - you'd know better. Maybe we can do that in a followup PR?

Reply:
sounds good, so we unblock this PR.